xsd/pmo/people.xsd

```diff
@@ -51,6 +51,7 @@ SOFTWARE.
       <xs:element name="reputation" type="xs:integer"/>
       <xs:element name="jobs" type="xs:integer"/>
       <xs:element name="projects" type="xs:integer"/>
+      <xs:element name="speed" type="xs:integer"/>
```

Comment:
@g4s8 should be `xs:decimal` type, let's restrict it to only positive values too